DOCUMENTATION.md

```diff
@@ -1947,20 +1950,41 @@ It will show something like:
     }
 }
 ```
+To add ACLS a json-file can be created from this template:
+
+```json
+[
+    {
+        "Resource":"/datasets/your.Dataset*",
+        "Action":"read",
+        "Deny":false
+    }
+]
+```
+After this file has been created it should be uploaded to the datahub.
```

Comment:
should we clarify what <client-id> is?

Reply:
This is specified just a few lines above, 1928